scripts/systemtap/kem/kemd.c

```diff
@@ -55,17 +54,17 @@ static void kemd_cleanup_module(void);
 static int kemd_open(struct inode *inode, struct file *filp);
 static int kemd_release(struct inode *inode, struct file *filp);
 static ssize_t kemd_read(struct file *filp, char *buf,
-			 size_t len, loff_t *offset);
+						 size_t len, loff_t *offset);
 static ssize_t kemd_write(struct file *filp, const char *buf,
-			  size_t len, loff_t *offset);
+						  size_t len, loff_t *offset);
```

Comment:
Please restore to the original code.

Reply:
It is looking good here, but we will come to know after checking in